docs/content_management/images/images.md

```diff
@@ -62,7 +62,7 @@ For more information, see the [bundle's documentation](https://symfony.com/bundl
 !!! caution "Code injection in image EXIF"
 
     EXIF metadata of an image may contain for example, HTML, JavaScript, or PHP code. 
-    [[= product_name =]] is itself doesn't parse EXIF metadata, but third-party bundles must be secured against this eventuality.
+    [[= product_name =]] itself doesn't parse EXIF metadata, but third-party bundles must be secured against this eventuality.
```

Comment:
@glye maybe something to include it in internal security training :D also possibly for the security checklist, or is that an overkill?

Reply:
Yes, could be indeed. Possibly both. There are other tricks with images, could mention that briefly. For example crafted flaws in images that exploit a vuln in GD. All file processing is potentially risky.

Reply:
I can make a suggestion here after demo.